### ISSUE TITLE
Does not Support My Locale Config

### ISSUE THREAD

**Issue by jnnkB** [hugolabe/Wike#200]
When I try to rune Wike, I get the following error:

```terminal
$ flatpak run com.github.hugolabe.Wike

(process:2): Gtk-WARNING **: 14:16:11.108: Locale not supported by C library.
	Using the fallback 'C' locale.
flatpak-spawn: Invalid byte sequence in conversion input
Try "flatpak-spawn --help" for more information.

** (wike:2): ERROR **: 14:16:11.645: readPIDFromPeer: Unexpected short read from PID socket
```

However like this, it works:

```terminal
$ LC_ALL=en_US.UTF-8 flatpak run com.github.hugolabe.Wike
```

How can I fix this, without overwriting all my locale?

```
locale
LANG=en_US.UTF-8
LC_CTYPE="en_US.UTF-8"
LC_NUMERIC=de_DE.UTF-8
LC_TIME=de_DE.UTF-8
LC_COLLATE="en_US.UTF-8"
LC_MONETARY=de_DE.UTF-8
LC_MESSAGES="en_US.UTF-8"
LC_PAPER=de_DE.UTF-8
LC_NAME="en_US.UTF-8"
LC_ADDRESS=de_DE.UTF-8
LC_TELEPHONE=de_DE.UTF-8
LC_MEASUREMENT=de_DE.UTF-8
LC_IDENTIFICATION="en_US.UTF-8"
LC_ALL=
```

**Comment by hugolabe** [comment]
Hi @jnnkB 
I have been able to reproduce the problem using the same locale configuration on my computer.

I have noticed that when using a mixed locale configuration (different _Language_ and _Formats_ selection in _Region & Language_) all flatpak applications show the warning `Locale not supported by C library` when starting them. Additionally, applications that use webkit fail to start, showing the error `readPIDFromPeer: Unexpected short read from PID socket`.

It seems to be a problem that affects flatpak applications that use webkit. The same happens with Epiphany or Newsflash.

A workaround I have found to avoid it is to use the parameter `--env=LC_ALL=en_US.UTF-8` in the flatpak command:

`flatpak run --env=LC_ALL=en_US.UTF-8 com.github.hugolabe.Wike`
